boards/arm/lpcxpresso55s69/lpcxpresso55s69_ns.dts

```diff
@@ -24,8 +24,9 @@
 	};
 
 	chosen {
-		zephyr,sram = &sram0;
+		zephyr,sram = &non_secure_ram;
 		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot1_partition;
```

Comment:
If we intend to support upgrades (which should work at least with the Zephyr driver), we'll also need slots 2 and 3 to hold the upgrade images of the two images.